service-monitor/src/main/java/com/yahoo/vespa/service/slobrok/SlobrokMonitorManagerImpl.java

```diff
@@ -30,14 +33,28 @@
     private final Object monitor = new Object();
     private final HashMap<ApplicationId, SlobrokMonitor> slobrokMonitors = new HashMap<>();
     private final DuperModelManager duperModel;
+    private final Transport transport;
+
+    private static int getTransportThreadCount() {
+        return Math.max(4, Runtime.getRuntime().availableProcessors());
+    }
 
     @Inject
     public SlobrokMonitorManagerImpl(DuperModelManager duperModel) {
-        this(SlobrokMonitor::new, duperModel);
+        this(new Transport("slobrok-monitor", getTransportThreadCount() / 4), duperModel);
     }
 
-    SlobrokMonitorManagerImpl(Supplier<SlobrokMonitor> slobrokMonitorFactory, DuperModelManager duperModel) {
+    private SlobrokMonitorManagerImpl(Transport transport, DuperModelManager duperModel) {
+        this(transport, new Supervisor(transport), duperModel);
```

Comment:
should the orb be tuned for small buffers?

Reply:
Perhaps. Ok.